invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/dashboard.html

```diff
@@ -15,16 +15,18 @@
 {%- endblock javascript %}
 
 {%- block page_body %}
-<div class="ui container dashboard-user-header-container">
-  <div class="ui items">
-    <div class="item">
-      <div class="ui tiny circular image">
-        <img src="/static/images/placeholder.png" />
-      </div>
-      <div class="content">
-        <span class="header">{{current_user.email}}</span>
-        <div class="meta">
-          <span>European Organization of Nuclear Research (CERN)</span>
+<div class="dashboard-header">
+  <div class="ui container">
+    <div class="ui items">
+      <div class="item">
+        <div class="ui image tinier circular circle">
+          <img src="/static/images/placeholder.png" />
+        </div>
+        <div class="content">
+          <span class="header">{{current_user.email}}</span>
+          <div class="meta">
+            <span>European Organization of Nuclear Research (CERN)</span>
```

Comment:
This should not be hardcoded! I missed in the previous PR sorry! Can you please remove it? I don't think we have the affilitation of the user yet.